spec/presenters/summary/html_presenter_spec.rb

```diff
@@ -5,80 +5,103 @@
-            Summary::Sections::IncomeDetails,
-            Summary::Sections::OtherIncomeDetails,
-            Summary::Sections::HousingPayments,
-            Summary::Sections::OtherOutgoingsDetails,
-            Summary::Sections::SupportingEvidence,
+      context 'for a "submitted" datastore application' do
+        let(:crime_application) { datastore_application }
+
+        let(:expected_sections) do
+          %w[
+            Overview
+            ClientDetails
+            ContactDetails
+            CaseDetails
+            Offences
+            Codefendants
+            NextCourtHearing
+            FirstCourtHearing
+            JustificationForLegalAid
+            PassportJustificationForLegalAid
+            EmploymentDetails
+            IncomeDetails
+            OtherIncomeDetails
+            HousingPayments
+            OtherOutgoingsDetails
+            SupportingEvidence
+            LegalRepresentativeDetails
           ]
-        )
+        end
+
+        it { is_expected.to match_array(expected_sections) }
       end
-      # rubocop:enable RSpec/ExampleLength
     end
 
-    context 'for a completed application (API response)' do
-      let(:crime_application) do
-        JSON.parse(LaaCrimeSchemas.fixture(1.0).read)
+    context 'when a PSE application' do
+      let(:application_type) { 'post_submission_evidence' }
+
+      context 'for an "in progress" database application' do
+        let(:crime_application) { database_application }
+        let(:expected_sections) { %w[ClientDetails SupportingEvidence] }
+
+        it { is_expected.to match_array(expected_sections) }
       end
 
-      # rubocop:disable RSpec/ExampleLength
-      it 'has the right sections in the right order' do
-        expect(
-          subject.sections
-        ).to match_instances_array(
-          [
-            Summary::Sections::Overview,
-            Summary::Sections::ClientDetails,
-            Summary::Sections::ContactDetails,
-            Summary::Sections::CaseDetails,
-            Summary::Sections::Offences,
-            Summary::Sections::Codefendants,
-            Summary::Sections::NextCourtHearing,
-            Summary::Sections::FirstCourtHearing,
-            Summary::Sections::JustificationForLegalAid,
-            Summary::Sections::PassportJustificationForLegalAid,
-            Summary::Sections::EmploymentDetails,
-            Summary::Sections::IncomeDetails,
-            Summary::Sections::OtherIncomeDetails,
-            Summary::Sections::HousingPayments,
-            Summary::Sections::OtherOutgoingsDetails,
-            Summary::Sections::SupportingEvidence,
-            Summary::Sections::LegalRepresentativeDetails,
-          ]
-        )
+      context 'for a "submitted" datastore application' do
+        let(:crime_application) { datastore_application }
+
+        let(:expected_sections) do
+          %w[Overview ClientDetails SupportingEvidence LegalRepresentativeDetails]
```

Comment:
Even though it is only three I think it makes it easier to parse if it is horizontally stacked like the tests above 👍